src/hypofuzz/dashboard.py

```diff
@@ -166,18 +273,19 @@ def start_dashboard_process(
     # we run a pytest collection step for the dashboard to pick up on databases
     # from custom profiles.
     _get_hypothesis_tests_with_pytest(pytest_args)
+    get_db()._db.add_listener(database_listener)
 
     # Ensure that we dump whatever patches are ready before shutting down
     def signal_handler(signum, frame):  # type: ignore
-        make_and_save_patches(pytest_args, LAST_UPDATE)
+        make_and_save_patches(pytest_args, REPORTS, METADATA)
```

Comment:
I think with our database-centric design now there's not much point writing out patches on signal or atexit.  If you can't think of anything either, let's delete this for simplicity?

Reply:
yeah I have "refactor patches" on my todo list which I think will involve removing this signal...we can do that now